modules/_labs/color-picker/react/lib/parts/SwatchBook.tsx

```diff
@@ -34,7 +34,7 @@ const SwatchContainer = styled('div')<SwatchContainerProps>(
 
     '&:focus': {
       outline: 'none',
-      ...focusRing(2, 2),
+      ...focusRing({width: 2, separation: 2}),
```

Comment:
Nit: You can remove `width: 2` everywhere. This is the default and it was only needed previously because you had to pass the width to specify the separation.